content/topics/python/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Python
+linkTitle: Python
+weight: 1
```

Comment:
This weight should match the weight on line 12 and should match the order we show the topics on the home page. So I believe this should be 4 and all the other weights should be adjusted...

(I'm not sure we did this right in all cases, so now is a good opportunity to fix/check.)

Reply:
Done and checked the others too.